detect_test.go

```diff
@@ -0,0 +1,233 @@
+package composer_test
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
```

Comment:
nit: can we please use the `os` package across this test/the buildpack instead?
`ioutil.WriteFile` -> `os.WriteFile`

Reply:
Thanks for the catch, I will do this. https://pkg.go.dev/io/ioutil